client/src/App.jsx

```diff
@@ -1,17 +1,124 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
+import ExpensesForm from "./components/ExpensesForm.jsx";
+import ExpenseList from "./components/ExpenseList.jsx";
+
 class App extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      lorem: ipsum
+      lorem: "ipsum",
```

Comment:
We can remove the placeholders once there's something here. I just drop lorem ipsum in to stop prettify from squishing things up.